tests/integration.rs

```diff
@@ -164,6 +178,19 @@ speculate::speculate! {
             assert!(!log.contains("shawl-child has started"));
         }
 
+        it "creates log file in custom dir with --log-dir" {
+            delete_log();
+            delete_log_custom_dir();
+
+            run_shawl(&["add", "--name", "shawl", "--log-dir", &log_custom_dir(), "--", &child()]);
+            run_cmd(&["sc", "start", "shawl"]);
+            run_cmd(&["sc", "stop", "shawl"]);
+
+            let log = std::fs::read_to_string(log_file_custom_dir()).unwrap();
+            assert!(!log.contains("shawl-child has started"));
```

Comment:
I was going to say the `!` shouldn't be there (the other test has it since it's for disabled logging), but it looks like this line isn't actually logged anywhere anyway 😅 I'll just switch it to look for `shawl-child message on stdout`.